modules/apps/rate-limiting/module.go

```diff
@@ -86,7 +86,7 @@ func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *r
 
 // GetTxCmd implements AppModuleBasic interface
 func (AppModuleBasic) GetTxCmd() *cobra.Command {
```

Comment:
Is this one actually mandatory to implement? I wonder if the CLI behaves correctly when returned as nil as well - should probably build locally and check that.